src/assertBounds.mli

```diff
@@ -10,13 +10,16 @@ module type SIG = sig
 
   (* The returned bool is true if the asserted bounds are not trivial
      (i.e. not implied by known bounds) *)
+  (** [var env x min max]
+      From an environment `env`, returns an environment `env'` in which the
+      bounds of `x` are updated to `min` and `max`. If the bounds were not
+      implied by other known bounds, the associated boolean will be `true`.
```

Comment:
please avoid conditionals with negation in documentation: it makes things much harder to understand. So in this case, state this as: "If the new bounds are already implied by other already present bounds, then returns the boolean 'false'" (and presumably in such cases the env is not actually changed I suppose ?)